docs/usage/workload_storage.rst

```diff
@@ -0,0 +1,104 @@
+Workload Storage
+================
+
+Any workload needs different volumes with different storage capacities
+to host its components. These volumes are stored in :term:`LVM LV`.
+
+Volumes
+-------
+
+Following the total storage capacity needed, establish a number of
```

Comment:
How does this follow the 'total' storage capacity required? The 'number of volumes' and their respective sizes are based on the workload needs. Total storage capacity required depends on this number.

Reply:
I think the sentence needs to be reworked. What you aim at, I guess, is something like

> Based on all required storage volumes, create a configuration as below: